### PR TITLE
Commented, re-structured and simplified the level builder

### PR DESCRIPTION
**Pull request by ramonfmir**
I've made some style changes, renamed some of the classes/methods to make them clearer and fixed some minor issues. Some unused code has been removed and the editor has been simplified to only generate obstacles. Last but not least, the controls are now clearer than ever for level designers :-)

controls

### CODE REVIEW

**Comment by ramonfmir** [comment]
This should fix #14 as well.

File: UnityProject/Assets/Editor/LevelBuilder/SceneView/GridVisualiser.cs [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+			lineScale, 
+			length * Constants.IsometricShiftX, 
+			lineScale);
+
+		// Rotate it facing towards the nearest point up left or up right. To
+		// understand the maths better, go to the IsometricPosition class.
+		Vector3 finalDirection = new Vector3(
+             (xAxis ? 1.0f : -1.0f) * Constants.IsometricShiftX, 
+             Constants.IsometricShiftY, 
+             1.0f);
```
Comment (OlafSzmidt) [inline UnityProject/Assets/Editor/LevelBuilder/SceneView/GridVisualiser.cs]: Fix indentation.

File: UnityProject/Assets/Editor/LevelBuilder/Util/EventTracker.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+/* Delegates what to do with the events. For now it only handles keyboard events
+ * but in the future it might be extended to mouse events.
+ */
+
+[InitializeOnLoad]
+public static class EventTracker
+{
+    static EventTracker()
+	{
+		//???
```
Comment (OlafSzmidt) [inline UnityProject/Assets/Editor/LevelBuilder/Util/EventTracker.cs]: Review this comment?

File: UnityProject/Assets/Editor/LevelBuilder/SceneView/GridVisualiser.cs (hunk 1 above)
Comment (OlafSzmidt) [inline UnityProject/Assets/Editor/LevelBuilder/SceneView/GridVisualiser.cs]: Fix indentation.

File: UnityProject/Assets/Editor/LevelBuilder/Util/AssetFetcher.cs [hunk 3]
```diff
@@ -6,63 +6,55 @@
...
-		return GetResurceNames ("Assets/Resources", ".png");
+		foreach (string name in GetLevels()) 
+			if (name == sceneName) 
+				return true;
+
+		return false;
 	}
 
-	public static IList<string> GetLevels()
+	public static IList<string> GetSprites()
```
Comment (OlafSzmidt) [inline UnityProject/Assets/Editor/LevelBuilder/Util/AssetFetcher.cs]: Is this called outside the class? Maybe we can make it private?

File: UnityProject/Assets/Editor/LevelBuilder/Util/AssetFetcher.cs [hunk 4]
```diff
@@ -6,63 +6,55 @@
...
 	{
-		return GetResurceNames ("Assets/Scenes", ".unity").Where(name => name != "Main").ToList();
+		return GetResourceNames("Assets/Resources", ".png");
 	}
 
-	private static bool IsScenePresent(string sceneName) 
+	public static IList<string> GetLevels()
```
Comment (OlafSzmidt) [inline UnityProject/Assets/Editor/LevelBuilder/Util/AssetFetcher.cs]: Is this called outside the class? Maybe(?) we can make it private.

**Pull request merged**